src/gui/widgets/callstacks.cc

```diff
@@ -22,10 +22,21 @@
 #include "core/callstacks.h"
 #include "core/psxemulator.h"
 #include "core/system.h"
+#include "supportpsx/memory.h"
 #include "fmt/format.h"
 #include "gui/gui.h"
 #include "imgui.h"
 
+static void drawSymbol(unsigned pc) {
+    std::pair<const uint32_t, std::string>* symbol = PCSX::g_emulator->m_cpu->findContainingSymbol(pc);
+    if (symbol) {
+        ImGui::SameLine();
+        ImGui::TextUnformatted(" :: ");
+        ImGui::SameLine();
+        ImGui::TextUnformatted(symbol->second.data(), &*symbol->second.end());
```

Comment:
That's actually not valid. You can't dereference `end()`, it causes an assert. I'm making changes on my end.